PanoramaPublic/test/src/org/labkey/test/tests/panoramapublic/TargetedMSTest.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2016-2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.labkey.test.tests.panoramapublic;
+
+import org.junit.BeforeClass;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestProperties;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.pages.targetedms.PanoramaDashboard;
+import org.labkey.test.util.APIContainerHelper;
+import org.labkey.test.util.ConfiguresSite;
+import org.labkey.test.util.DefaultSiteConfigurer;
+import org.labkey.test.util.LogMethod;
+import org.labkey.test.util.LoggedParam;
+import org.labkey.test.util.ReflectionUtils;
+import org.labkey.test.util.UIContainerHelper;
+import org.openqa.selenium.WebElement;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+public abstract class TargetedMSTest extends BaseWebDriverTest
```

Comment:
If we need a fork of this class, can you use a different class name to help avoid confusion?

Reply:
I don't need a fork.  Didn't realize I could extend targetedms.TargetedMSTest